unblob/handlers/filesystem/romfs.py

```diff
@@ -284,7 +284,7 @@ def create_inode(self, extract_root: Path, inode: FileHeader):
         elif inode.type == FS_TYPE.DIRECTORY:
             output_path.mkdir(mode=inode.mode, exist_ok=True)
         elif inode.type == FS_TYPE.FILE:
-            with output_path.open("wb") as f:
+            with output_path.open("xb") as f:
```

Comment:
just wondering why change it here?

also `output_path.write_bytes()` might be easier

Reply:
Answered above, you are right, that `write_bytes()` would be a simpler equivalent to current, but my change was not compatible with it - it would have succeeded only if the file does not exist.

Anyway, this change was reverted, it is not worth the time.